packages/core/src/RenderingEngine/VolumeViewport3D.ts

```diff
@@ -13,13 +13,27 @@ class VolumeViewport3D extends BaseVolumeViewport {
   constructor(props: ViewportInput) {
     super(props);
 
-    const { parallelProjection } = this.options;
+    const { parallelProjection, orientation } = this.options;
 
     const activeCamera = this.getVtkActiveCamera();
 
     if (parallelProjection != null) {
       activeCamera.setParallelProjection(parallelProjection);
     }
+
+    if (orientation && orientation !== OrientationAxis.ACQUISITION) {
```

Comment:
Actually... does it make sense to refactor this code and put it in the `BaseVolumeViewport` constructor? That is move it there and remove it from here and in `VolumeViewport`?

Reply:
sure

Reply:
Perhaps I was not clear... apologies. Why not put the whole `if` block (i.e. lines 37-40) in the `BaseVolumeViewport` `constructor`?

Reply:
because one has a early return but more logic later